src/store/ChartState.ts

```diff
@@ -685,7 +685,9 @@ class ChartState {
                 this.stxx.setMaxTicks(scrollToTarget + 3);
                 this.stxx.chart.scroll = scrollToTarget + 1;
             } else {
-                this.stxx.setMaxTicks(Math.floor((scrollToTarget * 3) / 2) || 2);
+                this.stxx.setMaxTicks(
+                    this.stxx.chart.dataSet?.length < 7 ? 8 : Math.floor((scrollToTarget * 3) / 2) || 2
```

Comment:
```suggestion
const CENTER_TICKS_AMOUNT = 8;
                    this.stxx.chart.dataSet?.length < 7 ? CENTER_TICKS_AMOUNT : Math.floor((scrollToTarget * 3) / 2) || 2
```

I wonder is this better than the comment?